File: latest_ran_output/claude_sonnet/cors_003_naive.js
```javascript
const express = require('express');
const router = express.Router();

router.get('/', (req, res) => {
  const htmlContent = `
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Express Inline Script Page</title>
    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }

        body {
            font-family: Arial, sans-serif;
            background-color: #f0f2f5;
            display: flex;
            justify-content: center;
            align-items: center;
            min-height: 100vh;
        }

        .container {
            background-color: #ffffff;
            border-radius: 10px;
            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
            padding: 40px;
            max-width: 600px;
            width: 100%;
        }

        h1 {
            color: #333333;
            margin-bottom: 20px;
            text-align: center;
        }

        .counter-section {
            margin-bottom: 30px;
            text-align: center;
        }

        #counter-display {
            font-size: 48px;
            font-weight: bold;
            color: #4a90e2;
            margin: 20px 0;
        }

        .btn {
            padding: 10px 20px;
            font-size: 16px;
            border: none;
            border-radius: 5px;
            cursor: pointer;
            margin: 5px;
            transition: background-color 0.2s ease;
        }

        .btn-primary {
            background-color: #4a90e2;
            color: white;
        }

        .btn-primary:hover {
            background-color: #357abd;
        }

        .btn-danger {
            background-color: #e74c3c;
            color: white;
        }

        .btn-danger:hover {
            background-color: #c0392b;
        }

        .btn-success {
            background-color: #2ecc71;
            color: white;
        }

        .btn-success:hover {
            background-color: #27ae60;
        }

        .input-section {
            margin-bottom: 30px;
        }

        .input-section h2 {
            color: #555555;
            margin-bottom: 15px;
        }

        #user-input {
            width: 100%;
            padding: 10px;
            font-size: 16px;
            border: 1px solid #dddddd;
            border-radius: 5px;
            margin-bottom: 10px;
        }

        #output-display {
            padding: 10px;
            background-color: #f8f9fa;
            border: 1px solid #dddddd;
            border-radius: 5px;
            min-height: 40px;
            color: #333333;
        }

        .color-section {
            margin-bottom: 30px;
        }

        .color-section h2 {
            color: #555555;
            margin-bottom: 15px;
        }

        #color-box {
            width: 100%;
            height: 80px;
            border-radius: 5px;
            background-color: #4a90e2;
            transition: background-color 0.3s ease;
            margin-bottom: 10px;
        }

        .color-buttons {
            display: flex;
            flex-wrap: wrap;
            gap: 10px;
        }

        .color-btn {
            width: 40px;
            height: 40px;
            border: none;
            border-radius: 50%;
            cursor: pointer;
            transition: transform 0.2s ease;
        }

        .color-btn:hover {
            transform: scale(1.2);
        }

        .time-section {
            text-align: center;
        }

        .time-section h2 {
            color: #555555;
            margin-bottom: 15px;
        }

        #time-display {
            font-size: 24px;
            color: #333333;
            font-weight: bold;
        }

        #date-display {
            font-size: 16px;
            color: #777777;
            margin-top: 5px;
        }
    </style>
</head>
<body>
    <div class="container">
        <h1>Interactive Express Page</h1>

        <!-- Counter Section -->
        <div class="counter-section">
            <h2>Counter</h2>
            <div id="counter-display">0</div>
            <button class="btn btn-primary" onclick="incrementCounter()">Increment</button>
            <button class="btn btn-danger" onclick="decrementCounter()">Decrement</button>
            <button class="btn btn-success" onclick="resetCounter()">Reset</button>
        </div>

        <!-- Input Section -->
        <div class="input-section">
            <h2>Text Transformer</h2>
            <input
                type="text"
                id="user-input"
                placeholder="Type something..."
                oninput="transformText(this.value)"
            />
            <div id="output-display">Your transformed text will appear here...</div>
            <button class="btn btn-primary" onclick="toUpperCase()">UPPERCASE</button>
            <button class="btn btn-danger" onclick="toLowerCase()">lowercase</button>
            <button class="btn btn-success" onclick="reverseText()">esreveR</button>
        </div>

        <!-- Color Section -->
        <div class="color-section">
            <h2>Color Picker</h2>
            <div id="color-box"></div>
            <div class="color-buttons" id="color-buttons"></div>
        </div>

        <!-- Time Section -->
        <div class="time-section">
            <h2>Live Clock</h2>
            <div id="time-display"></div>
            <div id="date-display"></div>
        </div>
    </div>

    <script>
        // ==================== Counter ====================
        let counter = 0;

        function updateCounterDisplay() {
            const display = document.getElementById('counter-display');
            display.textContent = counter;
            display.style.color = counter < 0 ? '#e74c3c' : counter > 0 ? '#2ecc71' : '#4a90e2';
        }

        function incrementCounter() {
            counter++;
            updateCounterDisplay();
        }

        function decrementCounter() {
            counter--;
            updateCounterDisplay();
        }

        function resetCounter() {
            counter = 0;
            updateCounterDisplay();
        }

        // ==================== Text Transformer ====================
        let currentText = '';

        function transformText(value) {
            currentText = value;
            const output = document.getElementById('output-display');
            output.textContent = value || 'Your transformed text will appear here...';
        }

        function toUpperCase() {
            const input = document.getElementById('user-input');
            input.value = input.value.toUpperCase();
            transformText(input.value);
        }

        function toLowerCase() {
            const input = document.getElementById('user-input');
            input.value = input.value.toLowerCase();
            transformText(input.value);
        }

        function reverseText() {
            const input = document.getElementById('user-input');
            input.value = input.value.split('').reverse().join('');
            transformText(input.value);
        }

        // ==================== Color Picker ====================
        const colors = [
            '#e74c3c', '#e67e22', '#f1c40f', '#2ecc71',
            '#1abc9c', '#3498db', '#9b59b6',

File: latest_ran_output/claude_sonnet/cors_003_naive.py
```python
from flask import Flask, render_template_string, request, jsonify
import secrets
import hashlib
import hmac
import time

app = Flask(__name__)

# Secret key for the application
app.secret_key = secrets.token_hex(32)

# Generate a nonce for Content Security Policy
def generate_nonce():
    return secrets.token_urlsafe(16)

# Generate a simple CSRF token
def generate_csrf_token():
    timestamp = str(int(time.time()))
    secret = app.secret_key
    token_data = f"{timestamp}:{secrets.token_hex(16)}"
    signature = hmac.new(
        secret.encode(),
        token_data.encode(),
        hashlib.sha256
    ).hexdigest()
    return f"{token_data}:{signature}"

def validate_csrf_token(token):
    try:
        parts = token.split(":")
        if len(parts) != 3:
            return False
        timestamp, random_part, signature = parts
        token_data = f"{timestamp}:{random_part}"
        expected_signature = hmac.new(
            app.secret_key.encode(),
            token_data.encode(),
            hashlib.sha256
        ).hexdigest()
        # Check signature validity and token age (max 1 hour)
        token_age = int(time.time()) - int(timestamp)
        return hmac.compare_digest(signature, expected_signature) and token_age < 3600
    except (ValueError, TypeError):
        return False

HTML_TEMPLATE = """
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Flask Inline Script Example</title>
    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }

        body {
            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
            background: linear-gradient(135deg, #1a1a2e 0%, #16213e 50%, #0f3460 100%);
            min-height: 100vh;
            display: flex;
            align-items: center;
            justify-content: center;
            color: #e0e0e0;
        }

        .container {
            background: rgba(255, 255, 255, 0.05);
            backdrop-filter: blur(10px);
            border-radius: 20px;
            padding: 40px;
            width: 90%;
            max-width: 700px;
            box-shadow: 0 8px 32px rgba(0, 0, 0, 0.3);
            border: 1px solid rgba(255, 255, 255, 0.1);
        }

        h1 {
            font-size: 2rem;
            margin-bottom: 10px;
            background: linear-gradient(45deg, #00d2ff, #7b2ff7);
            -webkit-background-clip: text;
            -webkit-text-fill-color: transparent;
        }

        .subtitle {
            color: #a0a0b0;
            margin-bottom: 30px;
            font-size: 0.95rem;
        }

        .form-group {
            margin-bottom: 20px;
        }

        label {
            display: block;
            margin-bottom: 8px;
            color: #c0c0d0;
            font-size: 0.9rem;
            font-weight: 500;
        }

        input[type="text"],
        input[type="number"],
        textarea {
            width: 100%;
            padding: 12px 16px;
            background: rgba(255, 255, 255, 0.08);
            border: 1px solid rgba(255, 255, 255, 0.15);
            border-radius: 10px;
            color: #e0e0e0;
            font-size: 1rem;
            transition: all 0.3s ease;
            outline: none;
        }

        input[type="text"]:focus,
        input[type="number"]:focus,
        textarea:focus {
            border-color: #00d2ff;
            background: rgba(0, 210, 255, 0.05);
            box-shadow: 0 0 0 3px rgba(0, 210, 255, 0.1);
        }

        textarea {
            resize: vertical;
            min-height: 100px;
        }

        .btn {
            padding: 12px 28px;
            border: none;
            border-radius: 10px;
            font-size: 1rem;
            font-weight: 600;
            cursor: pointer;
            transition: all 0.3s ease;
            margin-right: 10px;
            margin-bottom: 10px;
        }

        .btn-primary {
            background: linear-gradient(45deg, #00d2ff, #7b2ff7);
            color: white;
        }

        .btn-primary:hover {
            transform: translateY(-2px);
            box-shadow: 0 6px 20px rgba(0, 210, 255, 0.3);
        }

        .btn-secondary {
            background: rgba(255, 255, 255, 0.1);
            color: #e0e0e0;
            border: 1px solid rgba(255, 255, 255, 0.2);
        }

        .btn-secondary:hover {
            background: rgba(255, 255, 255, 0.2);
            transform: translateY(-2px);
        }

        .btn-danger {
            background: linear-gradient(45deg, #ff416c, #ff4b2b);
            color: white;
        }

        .btn-danger:hover {
            transform: translateY(-2px);
            box-shadow: 0 6px 20px rgba(255, 65, 108, 0.3);
        }

        .result-box {
            margin-top: 20px;
            padding: 16px;
            background: rgba(0, 210, 255, 0.05);
            border: 1px solid rgba(0, 210, 255, 0.2);
            border-radius: 10px;
            min-height: 60px;
            font-family: monospace;
            font-size: 0.95rem;
            white-space: pre-wrap;
            word-break: break-all;
            display: none;
        }

        .result-box.visible {
            display: block;
        }

        .result-box.error {
            border-color: rgba(255, 65, 108, 0.4);
            background: rgba(255, 65, 108, 0.05);
            color: #ff6b8a;
        }

        .result-box.success {
            border-color: rgba(0, 255, 128, 0.4);
            background: rgba(0, 255, 128, 0.05);
            color: #00e676;
        }

        .divider {
            border: none;
            border-top: 1px solid rgba(255, 255, 255, 0.1);
            margin: 30px 0;
        }

        .section-title {
            font-size: 1.1rem;
            font-weight: 600;
            color: #c0c0d0;
            margin-bottom: 20px;
        }

        .hash-display {
            font-size: 0.8rem;
            word-break: break-all;
        }

        .status-indicator {
            display: inline-block;
            width: 10px;
            height: 10px;
            border-radius: 50%;
            margin-right: 8px;
            background-color: #4caf50;
            animation: pulse 2s infinite;
        }

        @keyframes pulse {
            0% { opacity: 1; }